src/DataflowPass.cpp

```diff
@@ -29,68 +29,107 @@ void DataflowPass::setCFG(CFG *cfg) { this->cfg = cfg; }
 void DataflowPass::transfer(Instruction *instruction,
                             SetVector<Instruction *> workSet,
                             ProgramPoint &inputProgramPoint) {
-
+  // TODO: remove workset from arg list
```

Comment:
Can you expand this comment just a little bit to expand on why this is desirable. Just a "because ..." and then a reason is enough.

Reply:
`workSet` is not used. I'll just implement the change now